src/utils/map.hpp

```diff
@@ -8,20 +8,19 @@
 namespace nstd
```

Comment:
No touchy this file 😡

Reply:
needed it to run my program, didn't work without it, didn't mean to push it up though